monoio/src/builder.rs

```diff
@@ -40,7 +40,6 @@ impl<T> Default for RuntimeBuilder<T> {
     /// [monoio::blocking::BlockingStrategy] is to execute tasks on the local thread. In other
     /// words, there is no thread pool involved—all blocking I/O operations and heavy computations
     /// will block the current thread.
-    #[must_use]
```

Comment:
There's no side effect when create and then drop the builder. So this style is definitly a wrong way. `#[must_use]` can warn user if this happens.

Reply:
ok

Reply:
I'm sorry I just noticed the warning: `#[must_use]` has no effect when applied to a provided trait method.